libgraph/include/katana/PropertyGraph.h

```diff
@@ -997,6 +1010,38 @@ class KATANA_EXPORT PropertyGraph {
   Result<void> WriteView(
       const std::string& uri, const std::string& command_line);
 
+  /// Bitmask of nodes included in the transformation view. Should be used to construct arrow tables.
+  std::shared_ptr<arrow::Buffer> NodeBitmask() const noexcept {
+    if (is_transformed) {
+      auto length = original_to_transformed_nodes_.size();
```

Comment:
Shouldn't  you back these members by fields so that we don't create a new bitmask on every call (and a new `shared_ptr`)?

Reply:
It's a cheap operation that is also not on the hot path. Arrow buffer is just a fat pointer. This was my thinking for generating it on the fly. But I guess I can create it once at construction instead. Let me do that.